src/Styles/NewsletterForm.css

```diff
@@ -0,0 +1,26 @@
+.FormNL{
```

Comment:
maybe this should be separated into two files. Containers and Buttons files. The first should have the the .GreyRoundedContainer and the second the .GreyButton 

**Explanation why this separation is important**
Using this type of separation if in the future a programmer that did not work on the design of the program needs to use a style he easily knows where to look for the styles. If he needs a container style he will go to the containers file. If he needs a button style, he will go to the buttons file and he will not lost a lot of time looking around all the files looking for if exists a style like the one that he needs.

Reply:
i changed to a standardbutton name because its just a normal button with a margin left, dont have any other style, just a simply border just like a standard button